nabweb/views.py

```diff
@@ -571,18 +571,10 @@ def do_get_repository_info(repository, relpath):
             .strip()
             != ""
         )
-        info["tag"] = (
-            os.popen(
-                f"git -C {repo_dir} describe --long 2>/dev/null || "
-                f"git -C {repo_dir} describe --long --tags 2>/dev/null"
-            )
-            .read()
-            .strip()
-        )
         commits_count = (
             os.popen(
                 f"sudo -u \\#{repo_owner} "
-                f"git -C {repo_dir} fetch --no-tags >/dev/null && "
+                f"git -C {repo_dir} fetch --tags >/dev/null && "
```

Comment:
Removing the `--no-tags` option should be enough, to get the tags for the objects in the local (shallow) repository).
Adding `--tags` (to get **all** tags) seems a bit overkill, since older tags will not show up anyway (unless doing also unshallowing).